test/test_task/segmentation/test_model.py

```diff
@@ -31,3 +25,10 @@ def test_forward(self):
     data = Batch(pos=pos, x=x, batch=batch, y=y, coords=coords)
     model.set_input(data)
     model.forward()
+
+
+@pytest.mark.slow
```

Comment:
It's not perfect but it allows testing one epoch to see if the command is working. We can deactivate this test using the k command. maybe for each dataset, we should add a max sample variable.
```
 python -m pytest test -k "not slow"
```
It will be useful when we'll want to perform tests only locally.

Reply:
Pytorch lightning does a "validation" run at the beginning of the training for 2 batches, I wonder if there's a way we can just do that?

Reply:
I think this is possible in PTL https://forums.pytorchlightning.ai/t/running-validation-sanity-check-only/492